internal/util/k8s/configmap.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2025 The CephCSI Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package k8s
+
+import (
+	"context"
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// GetConfigMap retrieves a Kubernetes ConfigMap by its name and namespace.
+func GetConfigMap(configMapNamespace, configMapName string, ignoreNotFound bool) (*corev1.ConfigMap, error) {
```

Comment:
```suggestion
func GetConfigMap(namespace, name string, ignoreNotFound bool) (*corev1.ConfigMap, error) {
```

same here as well, dont pass `ignoreNotFound` as input